<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="enviarInicio_pushButton">
          <property name="text">
           <string>Agregar al inicio</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Capturar particulas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>velocidad</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>blue</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>destino x</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QSpinBox" name="green_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>origen en y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>green</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="origenY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>origen en x</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>red</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>destino y</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="Line" name="line_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QSpinBox" name="blue_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>color :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="origenX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="id_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>id</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="velocidad_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="destinoX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="destinoY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="mostrar_pushButton">
          <property name="text">
           <string>mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="4">
         <widget class="QPlainTextEdit" name="salida_plainTextEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="enviarFinal_pushButton">
          <property name="text">
           <string>Agregar al final</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="4">
         <widget class="QPushButton" name="mostrarTabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QTableWidget" name="table"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Buscar por ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="orderById_pushButton">
          <property name="text">
           <string>Ordenar ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="orderBySpeed_pushButton">
          <property name="text">
           <string>Ordenar velocidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Graficos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Grafo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" rowspan="7">
         <widget class="QGraphicsView" name="grafo_graphicsView"/>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="grafo_plainTextEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Origen X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="grafo_origenx_spinBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Origen Y</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="grafo_origeny_spinBox"/>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QPushButton" name="grafo_recorrer_pushButton">
          <property name="text">
           <string>Recorrer</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QPushButton" name="grafo_mostrar_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QPushButton" name="grafo_dibujar_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QPushButton" name="grafo_profundidad_pushButton">
          <property name="text">
           <string>Recorrido profundidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
